test/test_model.py

```diff
@@ -101,34 +99,15 @@ def test_prep():
         pass
     prophet.predict(X)
 
-    arima = ARIMA()
-    arima.predict(X)
-    arima._model = False
-    try:
-        arima.predict(X)
-    except ValueError:
-        # X_test needs to be either a pandas Dataframe with dates as the first column or an int number of periods for predict().
-        pass
-
-    lgbm = LGBM_TS(optimize_for_horizon=True, lags=1)
-    X = DataFrame(
-        {
-            "A": [
-                datetime(1900, 2, 3),
-                datetime(1900, 3, 4),
-                datetime(1900, 3, 4),
-                datetime(1900, 3, 4),
-                datetime(1900, 7, 2),
-                datetime(1900, 8, 9),
-            ],
-        }
-    )
-    y = np.array([0, 1, 0, 1, 0, 0])
-    lgbm.predict(X[:2])
-    lgbm.fit(X, y, period=2)
-    lgbm.predict(X[:2])
-    print(lgbm.feature_names_in_)
-    print(lgbm.feature_importances_)
+    # What's the point of callin ARIMA without parameters, or calling predict before fit?
```

Comment:
Is the inclusion of the commented code intended?

Reply:
It was meant as a remark to reviewers as to why the below lines in the test (calling ARIMA without parameters, and calling predict before fit) were removed. In my opinion both should just result in an exception.

Reply:
The code was there for code coverage. Removing will cause drop in code coverage.
Also, why is the LGBM_TS test removed?

Reply:
For code coverage, would it not make more sense to have another test in test_forecast.py that calls each TS model in turn on a tiny task? Likewise, I removed the lgbm_ts test as I felt it was replicating the tests made in test_forecast_py. Can put it back in if you like, but would it not make more sense to instead add the for-loop over all the TS models that the TSTask knows, in test_forecast.py, perhaps in a follow-up PR? :)

Reply:
Making a follow-up PR is OK. Don't decrease code coverage in this PR.

Reply:
Making a follow-up PR is OK. Don't decrease code coverage in this PR.

Reply:
Added a test in test_forecast.py that does a quick test fit for each of the available estimators. That fixes and improves coverage.

Reply:
Thanks. That's good. But the code below still covers code path which was not covered by the added code.